arrow/arrow.py

```diff
@@ -1258,6 +1258,13 @@ def humanize(
                 return locale.describe(granularity, delta, only_distance=only_distance)
 
             else:
+
+                if not granularity:
+                    raise ValueError(
+                        "Empty granularity list provided. "
+                        "Please select one or more from ['second', 'minute', 'hour', 'day', 'week', 'month', 'quarter', 'year']."
```

Comment:
It's probably better to write this as a plain english list rather than as a list. We should probably just join the member variable like we do here:
https://github.com/arrow-py/arrow/blob/2ade710d193b54663e4b3b7b85d2125d050be015/arrow/arrow.py#L1026-L1029

Reply:
I concur with this. Makes it easier to digest as an error.